mteb/tasks/Retrieval/eng/ClimateFEVERRetrievalFast.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class ClimateFEVERFast(AbsTaskRetrieval):
```

Comment:
(general comment) 

Fast seems a bit odd as it suggests eq. estimate just faster, it might be better to go with ClimateFEVERHardNegatives, ClimateFEVERHardLite, ClimateFEVERSlim. Feel free to come with another alternative as well. 

I would probably keep it in the same script as ClimateFEVER

Reply:
Great advice, moved each of them into the folder with their non hard negative task and renamed to HardNegatives